docs/src/main/sphinx/sql/show-create-function.md

```diff
@@ -2,20 +2,81 @@
 
 ## Synopsis
 
-```text
+```sql
```

Comment:
I think we use text for all others.. so leave it at that .. this is NOT valid SQL in that overview